website/sidebars.js

```diff
@@ -1,25 +1,11 @@
-module.exports = [[
-    {
-        type: 'category',
-        label: 'API Client',
-        items: [
-            'index',
-            'features',
-            'usage_concepts',
-        ],
-    },
-    // {
-    //     type: 'category',
-    //     label: 'Examples',
-    //     items: [
-    //         {
-    //             type: 'autogenerated',
-    //             dirName: 'examples',
-    //         },
-    //     ],
-    // },
-    {
-        type: 'doc',
-        id: 'changelog',
-    },
-]];
+module.exports = [
+    [
+        {
+            'type': 'category',
+            'label': 'API Client',
+            'items': [
```

Comment:
why the quotes on the keys? this should actually fail the linter i think

Reply:
No idea, fixed.